codewof/static/js/question_types/program.js

```diff
@@ -112,3 +118,51 @@ function run_python_code(user_code, test_case) {
         test_case.runtime_error = true;
     }
 }
+
+
+function setTutorialAttributes() {
+    $(".question-text").attr(
+        'data-intro',
+        'This is a description of what the code should do. It will tell you what your program should print for certain inputs.'
+    );
+    $("#python-editor").attr(
+        'data-intro',
+        "This is where you enter your code to solve the problem."
+    );
+    $("#run_code").attr(
+        'data-intro',
+        "Click 'Run code' to test your program against our test cases and see if they pass!"
+    );
+    $("#test-case-table").attr(
+        'data-intro',
+        "These are the test cases that will be run against your program."
+    );
+    // the first row in the test case table
+    $('#test-case-table tbody tr:nth-child(1)').attr(
+        'data-intro',
+        'Here is the first test case.'
+    );
+    // the input for the first test case
+    $('#test-case-table tbody tr:nth-child(1) td:eq(0)').attr(
+        'data-intro',
+        'Each line will be passed to your program as input for this particular test.'
```

Comment:
Maybe "If your program asks a user for input, each of these lines will be given in place of a real user." Not confident about the exact wording but that sort of style